### DISCUSSION TITLE
CoinFlip: A Smart Contract game built with Soroban and React

### DISCUSSION THREAD

**Discussion by simusud** [stellar/sorobounty-spectacular#32] [General]
Hello everyone,
I would like to submit a simple Smart Contract game: Coin Flip as my participation  for Sorobounty Spectacular.

- Source Code: https://github.com/simusud/coin-flip
- Youtube Video: https://www.youtube.com/watch?v=J-WjZYwKZ5M
- Tutorial: https://dev.to/simusud/coinflip-a-smart-contract-game-built-with-soroban-and-react-1mgj
- Live Web App: https://65721fb9c63121044e0ac4bf--jocular-toffee-e52918.netlify.app/

**Comment by briwylde08** [comment]
Hi there! Thank you so much for your submission. This program is not currently running, but Sorobounties is! Can you submit your project at https://github.com/stellar/sorobounties/discussions instead? Thank you!

**Reply by simusud** [reply]
Sure. Done !
